Backpack/Card/Classes/BPKDividedCard.m

```diff
@@ -101,6 +101,7 @@ - (void)setupViews {
     [self.contentView setUserInteractionEnabled:false];
     self.contentView.distribution = UIStackViewDistributionFillProportionally;
     self.contentView.alignment = UIStackViewAlignmentFill;
+    self.contentView.spacing = self.padded ? BPKSpacingBase : 0;
```

Comment:
This was a bug I found through testing - stack view spacing was only being applied by the `padded` setter, so `initWithPadded:YES` would result in `0` spacing

Reply:
Tests 🙌

Reply:
I can honestly say that every unit test I've ever written has either deepened my understanding of my own code or found a bug.

I don't know whether that say's good things about my testing abilities, or bad things about my coding abilities 😆

Reply:
Or bad things about how few tests I've written in my career 😆